unbound/metadata.csv

```diff
@@ -0,0 +1,83 @@
+unbound.thread.requestlist.overwritten,count,,request,,The number of requests in the request list that were overwritten by newer entries,1,unbound,thread.requestlist.overwritten
+unbound.thread.requestlist.exceeded,gauge,,query,,Queries that were dropped because the request list was full,1,unbound,thread.requestlist.exceeded
+unbound.thread.requestlist.current.all,gauge,,request,,Current size of the request list,0,unbound,thread.requestlist.current
+unbound.thread.requestlist.current.user,gauge,,request,,Current size of the request list (only the requests from client queries),0,unbound,thread.requestlist.current.user
+unbound.thread.recursion.time.avg,gauge,,second,,Average time it took to answer queries that needed recursive processing,0,unbound,thread.recursion.time.avg
+unbound.thread.recursion.time.median,gauge,,second,,The median of the time it took to answer queries that needed recursive processing,0,unbound,thread.recursion.time.median
+unbound.thread.tcpusage,gauge,,,,The currently held tcp buffers for incoming connections,0,unbound,thread.tcpusage
+unbound.total.num.queries,count,,query,,The total number of queries summed over threads,1,unbound,total.num.queries
+unbound.total.num.cachehits,count,,hit,,The total number of cache hits summed over threads,1,unbound,total.num.cachehits
+unbound.total.num.cachemiss,count,,miss,,The total number of cache misses summed over threads,1,unbound,total.num.cachemiss
+unbound.total.num.dnscrypt.crypted,count,,,,The total number of dnscrypt crypted summed over threads,1,unbound,total.num.dnscrypt.crypted
+unbound.total.num.dnscrypt.cert,count,,,,The total number of dnscrypt cert summed over threads,1,unbound,total.num.dnscrypt.cert
+unbound.total.num.dnscrypt.cleartext,count,,,,The total number of dnscrypt cleartext summed over threads,1,unbound,total.num.dnscrypt.cleartext
+unbound.total.num.dnscrypt.malformed,count,,,,The total number of dnscrypt malformed summed over threads,1,unbound,total.num.dnscrypt.malformed
+unbound.total.num.prefetch,count,,,,The total number of prefetch summed over threads,1,unbound,total.num.prefetch
+unbound.total.num.zero_ttl,count,,,,The total number of zero ttl summed over threads,1,unbound,total.num.zero_ttl
+unbound.total.num.recursivereplies,count,,,,The total number of recursive replies summed over threads,1,unbound,total.num.recursivereplies
+unbound.total.requestlist.avg,gauge,,request,,The total number of requests averaged over threads,0,unbound,total.requestlist.avg
+unbound.total.requestlist.max,gauge,,,,The maximum of the thread requestlist.max values,1,unbound,total.requestlist.max
+unbound.total.requestlist.overwritten,gauge,,,,The number of requests in the request list that were overwritten by newer entries summed over threads,1,unbound,total.requestlist.overwritten
+unbound.total.requestlist.exceeded,gauge,,,,The total number of queries that were dropped because the request list was full summed over threads,1,unbound,total.requestlist.exceeded
```

Comment:
```suggestion
unbound.total.requestlist.exceeded,gauge,,query,,The total number of queries that were dropped because the request list was full summed over threads,1,unbound,total.requestlist.exceeded
```